cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
```

Comment:
Yeah, the behavior pages like the ones Tanya mentioned could actually change as a result of this PR. This will effectively 'enable localization' on all of Pegasus pages by default (so 'enabling localization' for a page will no longer be necessary), so any pages that are (partially or completely) localized will start to show different content for different languages. If enabling localization could cause problems for any page, we might want to do an i18n pass on 'never enabled localization' pegasus pages after this gets merged to make sure there are no issues.